src/main.ts

```diff
@@ -70,14 +70,50 @@ const run = async (): Promise<void> => {
       return
     }
 
-    const comment = await createComment(octokit, owner, repo, issueNumber, result.commentBody)
+    if (!result.skipComment) {
+      const comment = await createComment(octokit, owner, repo, issueNumber, result.commentBody)
 
-    if (comment) {
-      core.setOutput('comment-created', 'true')
-      core.setOutput('comment-id', comment.id)
+      if (comment) {
+        core.setOutput('comment-created', 'true')
+        core.setOutput('comment-id', comment.id)
+
+        if (componentName) {
+          const componentComments = await getComponentComments(
+            octokit,
+            owner,
+            repo,
+            issueNumber,
+            componentName,
+          )
+
+          // Remove the comment with the same ID from componentComments
+          const filteredComments = componentComments.filter(
+            (componentComment) => componentComment.id !== comment.id,
+          )
+
+          const latestComment = filteredComments[0]
```

Comment:
Instead of just minimizing the latest comment, let's iterate this list and minimize all of them (maybe we can check if they're already minimized somehow?).